modules/network-observability-overview.adoc

```diff
@@ -14,8 +14,8 @@ You can customize the graphical view by using advanced options. To access the ad
 
 You can configure the details in the graph by using the *Display options* dropdown. The options available are:
 
-* *Metric type*: The metrics to be shown in graph. The default value is *Bytes*.
-* *Scope*: To choose the detail of components between which the network traffic flows. The default value is *Namespace*.
+* *Metric type*: The metrics to be shown in *Bytes* or *Packets*. The default value is *Bytes*.
+* *Scope*: To choose the detail of components between which the network traffic flows. You can set the scope to *Node*, *Namespace*, *Owner*, or *Resource*. *Owner* is an aggregation of resources. *Resource* can be a pod, node, in case of host-network traffic, or an unknown IP address. The default value is *Namespace*.
```

Comment:
thanks @snarayan-redhat , one missing thing: *resource* can also be a service. I'd list it in second position as it's a common one:
> *Resource* can be a pod, service, node in case of host-network traffic, or an unknown IP address.